app/views/layouts/listnav/_container_group.html.haml

```diff
@@ -60,3 +60,8 @@
             = link_to("#{ui_lookup(:table => "container_project")}: #{@record.container_project.name}",
                 {:controller => "container_project", :action => 'show', :id => @record.container_project.id.to_s},
                 :title => _("Show this container groups's parent %s") % ui_lookup(:table => "container_project"))
+        - if role_allows(:feature => "container_replicator") && !@record.container_replicator.nil?
+          %li
+            = link_to("#{ui_lookup(:table => "container_replicator")}: #{@record.container_replicator.name}",
+                {:controller => 'container_replicator', :action => 'show', :id => @record.container_replicator},
```

Comment:
Here in other places we're using `@record.container_replicator.id.to_s`, not sure what's the most correct. cc @matthewd @Fryguy @chessbyte